azure/packages/azure-client/package.json

```diff
@@ -62,7 +62,7 @@
 		"@fluidframework/azure-client-previous": "npm:@fluidframework/azure-client@2.0.0-internal.6.3.0",
```

Comment:
This PR only contains three kinds of changes:

1.  Bump build-tools to prerelease version "0.24.0-192663".
2.  Regenerated type test files.  (*.generated.ts)
3.  Removing type tests that now pass from 'package.json' and *.generated.ts.

All changes were performed by tools.  I'll "pre-review" the first two.